exist-core/src/main/java/org/exist/xquery/functions/util/Base64Functions.java

```diff
@@ -42,76 +42,80 @@
 /**
  * Base64 String conversion functions.
  *
- * @author  Andrzej Taramina (andrzej@chaeron.com)
+ * @author Andrzej Taramina (andrzej@chaeron.com)
  */
 
-public class Base64Functions extends BasicFunction
-{
-    protected static final Logger           logger       = LogManager.getLogger( Base64Functions.class );
-
-   public final static FunctionSignature signatures[] = {
-        new FunctionSignature(
-            new QName( "base64-encode", UtilModule.NAMESPACE_URI, UtilModule.PREFIX ),
-            "Encodes the given string as Base64",
-            new SequenceType[] {
-                new FunctionParameterSequenceType( "string", Type.STRING, Cardinality.ZERO_OR_ONE, "The string to be Base64 encoded" )
-            },
-            new FunctionReturnSequenceType( Type.STRING, Cardinality.ZERO_OR_ONE, "the Base64 encoded output, with trailing newlines trimmed" )
-        ),
-		
-		 new FunctionSignature(
-            new QName( "base64-encode", UtilModule.NAMESPACE_URI, UtilModule.PREFIX ),
-            "Encodes the given string as Base64",
-            new SequenceType[] {
-                new FunctionParameterSequenceType( "string", Type.STRING, Cardinality.ZERO_OR_ONE, "The string to be Base64 encoded" ),
-				new FunctionParameterSequenceType( "trim", Type.BOOLEAN, Cardinality.EXACTLY_ONE, "Trim trailing newlines?" )
-            },
-            new FunctionReturnSequenceType( Type.STRING, Cardinality.ZERO_OR_ONE, "the Base64 encoded output" )
-        ),
-		
-         new FunctionSignature(
-            new QName( "base64-decode", UtilModule.NAMESPACE_URI, UtilModule.PREFIX ),
-            "Decode the given Base64 encoded string back to clear text",
-            new SequenceType[] {
-                new FunctionParameterSequenceType( "string", Type.STRING, Cardinality.ZERO_OR_ONE, "The Base64 string to be decoded" )
-            },
-            new FunctionReturnSequenceType( Type.STRING, Cardinality.ZERO_OR_ONE, "the decoded output" )
-        )
+public class Base64Functions extends BasicFunction {
+    protected static final Logger logger = LogManager.getLogger(Base64Functions.class);
+
+    public final static FunctionSignature signatures[] = {
+            new FunctionSignature(
+                    new QName("base64-encode", UtilModule.NAMESPACE_URI, UtilModule.PREFIX),
+                    "Encodes the given string as Base64",
+                    new SequenceType[]{
+                            new FunctionParameterSequenceType("string", Type.STRING, Cardinality.ZERO_OR_ONE, "The string to be Base64 encoded")
+                    },
+                    new FunctionReturnSequenceType(Type.STRING, Cardinality.ZERO_OR_ONE, "the Base64 encoded output, with trailing newlines trimmed")
+            ),
+
+            new FunctionSignature(
+                    new QName("base64-encode", UtilModule.NAMESPACE_URI, UtilModule.PREFIX),
+                    "Encodes the given string as Base64",
```

Comment:
`Encodes the given string into the Base64 alphabet as defined by RFC 2045 §6.8 Base64 Content-Transfer-Encoding.`

Reply:
No benefit improving the description of deprecated functions